app/assets/stylesheets/layouts.css.scss

```diff
@@ -6,7 +6,7 @@
 }
 
 .header{
-  font-family: arial;
+  font-family: "Times New Roman";
```

Comment:
If you want to use TNR for the whole website (which you should, it's usually better to stick to one font), you should set in globally on your body. Since rails doesn't accept any css on the ```body``` element, you can create a ```.body``` class and put it there. add the class to the body in ```application.html.erb```